gbpservice/nfp/service_vendor_agents/vyos/agent/DEBIAN/control

```diff
@@ -0,0 +1,8 @@
+Package: vyos
+Source: vyos
+Version: 2.5
+Architecture: all
+Maintainer: One Convergence
+Section: devel
+Priority: optional
+Description: vyos package
```

Comment:
One thing I am not sure is if it will be allowed to submit the packaging stuff to opensource. If they allow, we need to find some references on where to keep this.